Packs/Gatewatcher-AionIQ/Integrations/GCenter103/GCenter103.py

```diff
@@ -0,0 +1,851 @@
+from typing import  (
+    Any
+)
+
+import demistomock as demisto  # noqa: F401 
+from CommonServerPython import *  # noqa: F401, F403 
+import urllib3
+import json
+import os
+import requests
+from datetime import datetime
+
+# Disable insecure warnings
+urllib3.disable_warnings()  # pylint: disable=no-member
+
+class GwAPIException(Exception):
+    """A base class from which all other exceptions inherit.
+
+    If you want to catch all errors that the gwapi_benedictine package might raise,
+    catch this base exception.
+    """
+
+class GwRequests:
+    """Allows to easily interact with HTTP server.
+
+    Class features:
+      - Get requests package wrapper.
+      - Put requests package wrapper.
+      - Post requests package wrapper.
+      - Delete requests package wrapper.
+    """
+
+    PROXIES = {
+        "http": "",
+        "https": ""
+    }
+
+    def __init__(self, ip: str, headers: dict = {}, check_cert: bool = False,
+                 proxy: bool = False) -> None:
+        """Init.
+
+        Disable urllib3 warning. Allow unsecure ciphers.
+
+        Args:
+            ip: IP address of the HTTP server.
+            check_cert: True to validate server certificate and False instead.
+            proxies: Requests proxies. Default to no proxies.
+        """
+        self.index_values = [
+            "suricata",
+            "codebreaker",
+            "malware",
+            "netdata",
+            "syslog",
+            "machine_learning",
+            "retrohunt",
+            "iocs"
+        ]
+        self.ip = ip
+        self.headers = headers
+        self.check_cert = check_cert
+        if proxy:
```

Comment:
You can use the `handle_proxies()` func from CommonServer which should also cover some edgecases.